pkg/server/plugin/upstreamauthority/vault/renewer.go

```diff
@@ -0,0 +1,37 @@
+package vault
+
+import (
+	"github.com/hashicorp/go-hclog"
+	vapi "github.com/hashicorp/vault/api"
+)
+
+type Renew struct {
+	Logger hclog.Logger
+}
+
+func NewRenew(logger hclog.Logger) *Renew {
```

Comment:
I'm ok with the original code here, i.e., passing everything needed to  construct the renewer in this function, I just thought maybe the logger could also be passed as a parameter instead of using a default logger.

Reply:
I understand the intent of the your comment.
I think there is no problem with passing all the argument including logger, so I'll fix it.

Reply:
fix: 8e74413